package-lock.json

```diff
@@ -52,9 +52,9 @@
       "dev": true
     },
     "jasmine-core": {
-      "version": "3.2.0",
-      "resolved": "https://registry.npmjs.org/jasmine-core/-/jasmine-core-3.2.0.tgz",
-      "integrity": "sha512-35OOToo2lFczwZ/FdJkUOO/Gsp9FW7viWChYA7OgBfpjgTxbxmNKyNrGS3HHREHay5nJwJvu4RqAlvcBcCAWeA==",
+      "version": "3.2.1",
+      "resolved": "http://npm-packages.rmdev.zone/jasmine-core/-/jasmine-core-3.2.1.tgz",
```

Comment:
Hmm, this doesn't seem right? Do you have a non-standard proxy set up or something?

Reply:
Yes we do - good spot, I didn't notice it either :)